### ISSUE TITLE
Windows 10 not hiding anyone

### ISSUE THREAD

**Issue by BelodedAleksey** [M00nRise/ProcessHider#18]
Tested different proccesses. Injection done, but no hiding.

**Comment by Wakeelfahmed** [comment]
Hi, did u find any solution?

**Comment by AhmadAmmar260** [comment]
Dear M00nRise,

I hope this message finds you well.

 I have been working with your ProcessHider project on GitHub. I am currently attempting to automate a process where I want an executable file to be run and immediately hidden with a specific name or process ID without any user interaction, command prompt, or additional steps.

Specifically, my challenge is to have the executable run and automatically change its process name to a name I specify in the code or lock it to a specific process ID (PID) once it is executed. The process should be hidden immediately without requiring any visible CMD windows or other prompts.

If you could kindly provide any guidance on how to achieve this, or if there is an existing feature within ProcessHider that would allow for this functionality, I would greatly appreciate your help.

Hope for reply...

Thank you for your time and for your contributions to this project. I look forward to your response.

Best regards,

**Comment by madhead341** [comment]
yeah, you can do this by hardcoding a few process names into a python script. PIDs aren’t stable so you’ll need to track the one you want after launching the exe.

you can use ``CreateProcess`` with ``CREATE_NO_WINDOW`` so the process runs silent. then hook into taskmgr.exe and hide the PID from there.
